lighthouse-cli/test/fixtures/oopif.html

```diff
@@ -5,6 +5,6 @@
   </head>
   <body>
     <h1>Hello frames</h1>
-    <iframe name="oopif" src="https://airhorner.com"></iframe>
+    <iframe name="oopif" src="https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/" style="width: 100vw; height: 100vh;"></iframe>
```

Comment:
also, should we just be making our own OOPIF here rather than relying on this page having another nested one? We could do a page on the other port to go cross origin (assuming that triggers OOP) and have that embed the youtube video, for instance

Reply:
I wish, I tried like 15 different combinations but localhost never seems to be oopif'd on its own, so we need at least one publicly hosted page that then also iframes a google-y origin that will guarantee 2 separate processes beyond our localhost one